bff-tiltaksarrangor/src/main/kotlin/no/nav/amt/tiltak/bff/tiltaksarrangor/VeilederController.kt

```diff
@@ -112,8 +112,7 @@ class VeilederController (
 		arrangorAnsattTilgangService.verifiserHarRolleAnywhere(ansatt.id, ArrangorAnsattRolle.VEILEDER)
 
 		val deltakerliste = arrangorVeilederService.hentDeltakerliste(ansatt.id).filter {
-			val arrangorId = gjennomforingService.getArrangorId(it.gjennomforingId)
-			arrangorAnsattTilgangService.harRolleHosArrangor(ansatt.id, arrangorId, ArrangorAnsattRolle.VEILEDER)
+			arrangorAnsattTilgangService.harRolleHosArrangor(ansatt.id, it.arrangorId, ArrangorAnsattRolle.VEILEDER)
```

Comment:
Hvis vi ser at det blir behov for det kan vi effektivisere ennå mer her ved å trekke ut alle unike arrangører fra deltakerlisten slik at vi kun sjekker tilgang en gang pr arrangør, men enn så lenge lot jeg det være sånn for å beholde lesbarheten.